package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@ipld/unixfs",
   "description": "An implementation of the UnixFS in JavaScript designed for use with multiformats",
-  "version": "2.1.2",
+  "version": "3.0.0",
```

Comment:
Please revert - this will be done automatically by CI when the module is released.